### DISCUSSION TITLE
XIAO ESP32S3 Project MicroPython | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1947] [Q&A]
# XIAO ESP32S3 Project MicroPython | Seeed Studio Wiki

MicroPython for XIAO ESP32S3 Sense

https://wiki.seeedstudio.com/XIAO_ESP32S3_Micropython/

**Comment by milandahal213** [comment]
Is there any documentation on how to set different frame sizes and configurations ? The disabled help command is not very helpful :)

**Comment by Harrison-Xu** [comment]
Hello, during test I got an error. 

On the client (computer) side: 
<img width="894" alt="企业微信截图_1e33e0c3-da76-419a-a0dd-3229cfea9bf4" src="https://github.com/user-attachments/assets/8772d6a6-2fb4-4c33-be72-30721a091e8e" />

On the server (XIAO) side: 
<img width="1072" alt="企业微信截图_1eca4df4-8550-4a05-b444-86086304d966" src="https://github.com/user-attachments/assets/bfdadb0d-0c17-4b4c-b5c4-f1303dda33b9" />

Could anyone help me with solving this `TCP send error object with buffer protocol required` error? Many thanks in advance!

**Comment by YWS0915** [comment]
為什麼OV3660的無法初始化?
